scripts/templates/trcddi.cpp.mako

```diff
@@ -49,6 +49,10 @@ namespace ur_tracing_layer
 
         context.notify_end(${th.make_func_etor(n, tags, obj)}, "${th.make_func_name(n, tags, obj)}", &params, &result, instance);
 
+        std::ostringstream args_str;
+        ur::extras::printFunctionParams(args_str, ${th.make_func_etor(n, tags, obj)}, &params);
+        context.logger.info("---> ${th.make_func_name(n, tags, obj)}({}) -> {};", args_str.str(), result);
```

Comment:
The `--->` at the start here is to make the formatting a little more like the SYCL_PI_TRACE equivalent, which makes it easier to pick out individual calls. This does differ from the `urtrace` collector though, so we might not want to do this. I don't have strong feelings either way.

Reply:
I don't have a preference either way.